cluster/gce/gci/configure-helper.sh

```diff
@@ -0,0 +1,749 @@
+  if echo "${ADMISSION_CONTROL:-}" | grep -q "LimitRanger"; then
+    setup-addon-manifests "admission-controls" "limit-range"
+  fi
+
+  # Place addon manager pod manifest.
+  cp "${src_dir}/kube-addon-manager.yaml" /etc/kubernetes/manifests
+}
+
+# Starts a fluentd static pod for logging.
+function start-fluentd {
+  echo "Start fluentd pod"
+  if [[ "${ENABLE_NODE_LOGGING:-}" == "true" ]]; then
+    if [[ "${LOGGING_DESTINATION:-}" == "gcp" ]]; then
+      cp "${KUBE_HOME}/kube-manifests/kubernetes/fluentd-gcp.yaml" /etc/kubernetes/manifests/
+    elif [[ "${LOGGING_DESTINATION:-}" == "elasticsearch" ]]; then
+      cp "${KUBE_HOME}/kube-manifests/kubernetes/fluentd-es.yaml" /etc/kubernetes/manifests/
+    fi
+  fi
+}
+
+function reset-motd {
+  # kubelet is installed both on the master and nodes, and the version is easy to parse (unlike kubectl)
+  local -r version="$(/usr/bin/kubelet --version=true | cut -f2 -d " ")"
+  # This logic grabs either a release tag (v1.2.1 or v1.2.1-alpha.1),
+  # or the git hash that's in the build info.
+  local gitref="$(echo "${version}" | sed -r "s/(v[0-9]+\.[0-9]+\.[0-9]+)(-[a-z]+\.[0-9]+)?.*/\1\2/g")"
+  local devel=""
+  if [[ "${gitref}" != "${version}" ]]; then
+    devel="
+Note: This looks like a development version, which might not be present on GitHub.
+If it isn't, the closest tag is at:
+  https://github.com/kubernetes/kubernetes/tree/${gitref}
+"
+    gitref="${version//*+/}"
+  fi
+  cat > /etc/motd <<EOF
+
+Welcome to Kubernetes ${version}!
+
+You can find documentation for Kubernetes at:
+  http://docs.kubernetes.io/
+
+You can download the build image for this release at:
+  https://storage.googleapis.com/kubernetes-release/release/${version}/kubernetes-src.tar.gz
+
+It is based on the Kubernetes source at:
+  https://github.com/kubernetes/kubernetes/tree/${gitref}
+${devel}
+For Kubernetes copyright and licensing information, see:
+  /home/kubernetes/LICENSES
+
+EOF
+}
+
+
+########### Main Function ###########
+echo "Start to configure instance for kubernetes"
+
+KUBE_HOME="/home/kubernetes"
+if [[ ! -e "${KUBE_HOME}/kube-env" ]]; then
+  echo "The ${KUBE_HOME}/kube-env file does not exist!! Terminate cluster initialization."
+  exit 1
+fi
+
+source "${KUBE_HOME}/kube-env"
+config-ip-firewall
+create-dirs
+if [[ "${KUBERNETES_MASTER:-}" == "true" ]]; then
+  mount-master-pd
+  create-master-auth
+  create-master-kubelet-auth
+else
+  create-kubelet-kubeconfig
+  create-kubeproxy-kubeconfig
+fi
+
+assemble-docker-flags
+load-docker-images
+start-kubelet
+
+if [[ "${KUBERNETES_MASTER:-}" == "true" ]]; then
+  compute-master-manifest-variables
+  start-etcd-servers
+  start-kube-apiserver
+  start-kube-controller-manager
+  start-kube-scheduler
+  start-kube-addons
+else
+  start-kube-proxy
+  # Kube-registry-proxy.
+  if [[ "${ENABLE_CLUSTER_REGISTRY:-}" == "true" ]]; then
+    cp "${KUBE_HOME}/kube-manifests/kubernetes/kube-registry-proxy.yaml" /etc/kubernetes/manifests
+	fi
+fi
+reset-motd
```

Comment:
This should be the last thing that we do. Why do you configure fluentd after putting up the message?